ros2topic/ros2topic/verb/info.py

```diff
@@ -30,6 +31,21 @@ def add_arguments(self, parser, cli_name):
     def main(self, *, args):
         with DirectNode(args) as node:
             topic_name = args.topic_name
+            type_name = "NOT FOUND!"
+
+            all_topic_names_and_types = get_topic_names_and_types(node=node, include_hidden_topics=True)
+            for topic, type in all_topic_names_and_types:
```

Comment:
Please don't use the name of a builtin as a variable name.

Reply:
Recommend `type` -> `type_` per [PEP 8](https://www.python.org/dev/peps/pep-0008/#id36)

> single_trailing_underscore_: used by convention to avoid conflicts with Python keyword [...]